navfn/src/navfn_ros.cpp

```diff
@@ -80,6 +80,7 @@ namespace navfn {
       private_nh.param("planner_window_x", planner_window_x_, 0.0);
       private_nh.param("planner_window_y", planner_window_y_, 0.0);
       private_nh.param("default_tolerance", default_tolerance_, 0.0);
+      private_nh.param("plan_lenght_threshold", plan_lenght_threshold_, 50.0);
```

Comment:
The default behavior should be that there is no threshold, which is the current behavior.

Reply:
Changed the default value to 0 and added a check to not do anything in that case.